helpers.lua

```diff
@@ -136,6 +137,19 @@ function helpers.uformat(array, key, value, unit)
 end
 -- }}}
 
+-- {{{ Format seconds to HH:MM:SS
+function helpers.sec2hhmmss(seconds)
+    return ("%02d:%02d:%02d"):format(math.floor(seconds / 3600),
+                                     math.floor(seconds % 3600 / 60),
+                                     math.floor(seconds % 60))
+end
+
+-- {{{ Format seconds to MM:SS
+function helpers.sec2mmss(seconds)
+    return ("%02d:%02d"):format(math.floor(seconds / 60),
+                                math.floor(seconds % 60))
```

Comment:
I think skipping digits that are zero i.e. hours and minutes as sketched by https://github.com/vicious-widgets/vicious/issues/66#issuecomment-447276202 make sense to save space. (most songs will take less then an hour)

Reply:
Indeed, although elapsed and duration should have same format.